.travis.yml

```diff
@@ -17,8 +17,8 @@ env:
   global:
     - GRADLE_OPTS=-Dorg.gradle.daemon=false
   matrix:
-    - TEST_SUITE=checkstyle
     - TEST_SUITE=check OPTIONS=modernizer
+    - TEST_SUITE=checkstyle
```

Comment:
I would definitely execute checkstyle first. As it will always fail asap. So you already know that you have to fix this again

Reply:
They are not guaranteed to run in the specified order (but are started at almost the same time anyway in most cases). Thus it is more a display thing, and the main build is definitely the more important one.